crates/sui-types/src/crypto.rs

```diff
@@ -1457,7 +1457,8 @@ mod bcs_signable {
 
     pub trait BcsSignable: serde::Serialize + serde::de::DeserializeOwned {}
     impl BcsSignable for crate::committee::Committee {}
-    impl BcsSignable for crate::committee::CommitteeWithNetAddresses {}
+    impl BcsSignable for crate::committee::CommitteeWithNetworkMetadata {}
+    impl BcsSignable for crate::committee::NetworkMetadata {}
```

Comment:
Do these need to be signable?

Reply:
not any longer. Removed in the new commit